### DISCUSSION TITLE
Latest diagonal issues creating negative IBNR

### DISCUSSION THREAD

**Discussion by AuldKingCole** [casact/chainladder-python#233] [Q&A]
No matter what data I use to set up a triangle, my fitted models keep producing latest diagonals with what almost appear to be cumulative values increasing into past origins. The ultimate estimator and full triangle estimator are all working just fine, but the inaccurate latest diagonal and therefore the IBNR estimates are off:

`
sample=full_tri.loc[('private','bi_tort','Total'),'incurred']

sequence = [
    ("simple_dev", cl.Development(average="simple")),
    ("inverse_power_tail", cl.TailCurve(curve="inverse_power")),
]

pipe = cl.Pipeline(steps=sequence).fit(sample)

fitted=pipe.fit_transform(sample)
model=cl.Chainladder().fit(fitted)
`
The original triangle's latest diagonal:
![image](https://user-images.githubusercontent.com/94124555/141995249-0c6ac945-42c2-48c7-a2f4-63aee1953008.png)
The model's latest diagonal:
![image](https://user-images.githubusercontent.com/94124555/141995312-536f1b2d-fc78-4082-a87e-c61f1d64543e.png)

Thanks for any insight on the issue.

**Comment by jbogaardt** [comment]
I suspect its confused on whether your triangle `is_cumulative` or not. Specifically, it thinks it is incremental when in fact it is cumulative.

When creating your triangle, you can explicitly  let it know it is cumulative by setting `cumulative=True`   Refer to [this section](https://chainladder-python.readthedocs.io/en/latest/triangle.html#other-parameters) in the docs for more details.  I believe this should resolve your issue.

Truthfully, I think we need better handling of this that more loudly tells the user what is going on.  Currently, the `cumulative` argument is optional, but a lot of the functionality of the library needs to know whether the triangle is cumulative or incremental.
